site/configure.xml

```diff
@@ -497,71 +702,87 @@ CONFIG_FILE=/etc/rabbitmq/testdir/bunnies</pre>
           </tr>
           <tr>
             <td>
-              <code>tcp_listeners</code>
+              <code>listeners</code>
             </td>
             <td>
-              List of ports on which to listen for AMQP connections
-              (without SSL). Can contain integers (meaning "listen on
-              all interfaces") or tuples such as <code>{"127.0.0.1",
-              5672}</code> to listen on one or more interfaces.
+              Ports or hostname/pair on which to listen for AMQP connections
+              (without TLS). See the <a href="/networking.html">Networking guide</a> for more
+              details and examples.
 
-              <p>Default: <code>[5672]</code></p>
+              <p>Default: <code>listeners.tcp.default = 5672</code></p>
```

Comment:
The listeners configuration (for RabbitMQ/AMQP and the plugins too) is very confusing.

1. `tcp` or `ssl` below make no sense (I know it comes from the classic format). TLS is still over TCP and currently we support TCP or HTTP as the transport layer and this is orthognal to TLS. Can we use the protocol name or purpose instead (see example below)?
2. What is `default` in `listeners.tcp.default`?
3. If I compare the configuration for AMQP(S) and the management plugin:
 * `listeners` vs. `listener` (plural vs. singular)
 * `...tcp.default = 5672` vs. `...port = 15672`
 * `listeners.ssl = none` vs. `management.listener.ssl = true` (`none`/`true`)
 * `ssl_options` vs. `ssl_opts`

Below, I'm giving an example of a centralized configuration for listeners, with the following additional changes:
* I put the acceptors count and TCP options as leaves there as well.
* I use `tls` instead of `ssl` to reflect the naming used in the documentation.
* `listeners` may be too long; `listen` would be enough.

```
listen.amqp.port = 5672
listen.amqp.tls = true # Or "tls.enabled = true" if an intermediate node can't have a value
listen.amqp.tls.port = 5671
listen.amqp.tls.certfile = /path/to/cert.pem
listen.mqtt.port = 1883
listen.mqtt.tls = false
listen.mqtt.tls.port = 8883 # Ignored because TLS is disabled
listen.management.port = 15672
listen.management.tls = true
listen.management.tls.port = 15671

# For $protocol over HTTP:
listen.stomp.http.port = ...
listen.stomp.http.tls.port = ...
```

Default/common options could be set under a `default` node:
```
listen.default.nb_acceptors = 1
listen.default.handshake_timeout = 10000

listen.default.tls.enabled = true
listen.default.tls.certfile = /path/to/cert.pem
listen.default.tls.keyfile = /path/to/key.pem
listen.default.tls.nb_acceptors = 2 # Otherwise defaults to listen.default.nb_acceptors
```

This lacks an example of how to listen on specific interfaces.

What do you think?